.gitignore

```diff
@@ -28,8 +28,51 @@ override.tf.json
 .terraformrc
 terraform.rc
 
-# Lambda directories
-builds/
```

Comment:
`builds/` is used by the Lambda module.
`functions/pytest.ini` (or `.envrc`) may contain secrets.

Probably they worth to be ignored.

Reply:
<img  alt="image" src="https://media2.giphy.com/media/QMHoU66sBXqqLqYvGO/200.gif">

Reply:
re-added in https://github.com/terraform-aws-modules/terraform-aws-notify-slack/pull/152/commits/e091ea237fafd45d39ea5cb627e00877f3c6e55f